inst/doc/shared_cache.Rmd

```diff
@@ -1,52 +1,191 @@
+## Shared Cache {.tabset .tabset-fade .tabset-pills}
+
+Alison Appling, July 31, 2018;  
+Jake Zwart, February 4, 2019
+
 ---
-title: "Shared cache"
-author: "Alison Appling"
-date: "July 31, 2018"
 output: rmarkdown::html_vignette
 vignette: >
   %\VignetteIndexEntry{Shared cache}
   %\VignetteEngine{knitr::rmarkdown}
   \usepackage[utf8]{inputenc}
 ---
+<head>
+<style>
+data {color: #CA3542;}
+ind {color: #FEBC38;}
+build {color: #697F98;} 
+target {color: #37AFA9}
+</style>
+</head>
+
   
 ```{r setup, include=FALSE}
 knitr::opts_chunk$set(echo = TRUE, cache=FALSE, collapse=TRUE)
-knitr::opts_knit$set(root.dir = tempdir())
 ```
 
-## Setup
 
-```{r libs, message=FALSE}
-library(scipiper)
-library(dplyr)
-```
+### Terms and concept
 
-## Shared cache
+####Terms
```

Comment:
Should we include a definition of the yaml file here?

Reply:
Yeah I think that would be helpful. I can add a short description